### PR TITLE
DC-3353

### PR DESCRIPTION
**Pull request by jim-collins**

### CODE REVIEW

File: test/connectors/SecureMessageConnectorSpec.scala [hunk 1]
```diff
@@ -94,6 +94,55 @@ class SecureMessageConnectorSpec extends PlaySpec with MockitoSugar {
...
+
+      private val localDate = LocalDate.now()
+      private val letter = Letter("MRN 123", "CDS message", None, Sender("HMRC", localDate), None)
+      when(
+        mockHttpClient
+          .GET[Letter](any[String], any[Seq[(String, String)]], any[Seq[(String, String)]])(
+            any[HttpReads[Letter]],
+            any[HeaderCarrier],
+            any[ExecutionContext]))
+        .thenReturn(Future(letter))
```
Comment (alexflav23) [inline test/connectors/SecureMessageConnectorSpec.scala:140]: Could use `Future.successful`??

Comment (jim-collins) [reply test/connectors/SecureMessageConnectorSpec.scala:140]: Thanks I will change it

File: app/models/Letter.scala [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+ */
+
+package models
+
+import org.joda.time.{ DateTime, LocalDate }
+import play.api.libs.json.{ Format, Json }
+import play.api.libs.json.JodaReads.{ jodaDateReads, jodaLocalDateReads }
+import play.api.libs.json.JodaWrites.{ jodaDateWrites, jodaLocalDateWrites }
+
+@SuppressWarnings(Array("org.wartremover.warts.All"))
```
Comment (elabeca) [inline app/models/Letter.scala:24]: you can exclude the ones that are failing as opposed to all

**Review by muralishub — APPROVED** [review]

File: test/connectors/SecureMessageConnectorSpec.scala [hunk 3]
```diff
@@ -94,6 +94,55 @@ class SecureMessageConnectorSpec extends PlaySpec with MockitoSugar {
     }
   }
 
+  "SecureMessgaeConnector.getConversationContent" must {
```
[the remark this answers is not in the document]
Comment (jim-collins) [reply test/connectors/SecureMessageConnectorSpec.scala:97]: I will change it

**Review by chandankumarray — APPROVED** [review]

**Pull request merged**